olive/model/hf_utils.py

```diff
@@ -27,7 +27,7 @@ class HFComponent(ConfigBase):
     dummy_inputs_func: Union[str, Callable]
 
 
-class HFModelLoadingArgs(ConfigBase):
+class HFModelLoadingArgs(ConfigWithExtraArgs):
```

Comment:
Do we have any other use case than `HFModelLoadingArgs`?

Reply:
Yes, my qlora pass uses this feature too https://github.com/microsoft/Olive/blob/jambayk/qlora-dev/olive/passes/pytorch/qlora.py#L35. 

I can envision this being useful in the future too. Since, it sometimes feels harder to pass kwargs as a dict. Since we need to know what are defined fields and what are extra args.